native/src/main/java/io/ballerina/stdlib/http/api/client/actions/HttpClientAction.java

```diff
@@ -202,6 +254,34 @@ private static Map<String, Object> getPropertiesToPropagate(Environment env) {
         return subMap;
     }
 
+    private static BString makeRequestPath(BArray pathArray, BMap params) {
```

Comment:
IMO better name for this function would be `constructRequestPath`.

Reply:
Fixed via [8ebeff1](https://github.com/ballerina-platform/module-ballerina-http/pull/1148/commits/8ebeff1c68c5f668e6feabb4c0c1d113f75cb95d)